app/api/case/controller.js

```diff
@@ -14,11 +14,11 @@ const writeToGCSBucket = require('../../lib/writeToGCSBucket');
 /**
  * @method fetchCasePoints
  *
- * Returns all points of concern for the provided case.
+ * Returns all points of concern for the provided cases.
  *
  */
 exports.fetchCasePoints = async (req, res) => {
-  const { caseId } = req.query;
+  const { caseId } = req.body;
```

Comment:
We were previously pulling this info from query params but have since updated the API spec to pass this data in the body.